tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -91,10 +92,18 @@ static string GetSmName(std::pair<int, int> compute_capability) {
       {{7, 5}, 75},
   });
   int sm_version = 35;
-  auto it = m->find(compute_capability);
-  if (it != m->end()) {
-    sm_version = it->second;
-  } else {
+  // If the current compute capability isn't known, fallback to the
+  // most recent version before the unknown version.
+  for (auto iter = m->begin(); iter != m->end(); ++iter) {
+    auto k = iter->first;
+    if ((k.first < compute_capability.first) ||
```

Comment:
Unnecessary parens.

Reply:
It looks like a good opportunity for using `std::find_first_of()`

Reply:
I would need a find_last_of() with a predicate that does <= instead of ==. If you know a function that does the equivalent of an find_last_of(), tell me.

Reply:
Right. On the second though this whole thing is overly complicated for what it does. Why not use something as simple as this:

```
static int f(std::pair<int, int> cc) {
  int ccv = cc.first * 10 + cc.second;
  int best = 0;
  for (int v : {75, 72, 70, 62, 61, 60, 53, 52, 50, 37, 35}) {
      if (v <= ccv)
         return v;
  }
  return 35;
}
```

Reply:
I wanted to minimize the code change, but here it is.